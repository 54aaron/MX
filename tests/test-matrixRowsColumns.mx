int main(){
    Matrix m;
    Matrix n;
    int colsm;
    int rows;
    m = [[2,3,4],[6,4,5],[7,8,1],[9,7,8]];
    n = [[1,2,4],[3,4,5]];
    colsm = numCols(m);
    rows  = numRows(n);
    prints("==============================================");
    print_matrix(m);
    prints("==============================================");
    prints("Number of columns for m are: ");
    print(colsm);
    prints("==============================================");
    print_matrix(n);
    prints("==============================================");
    prints("Number of rows for n are: ");
    print(rows);
    prints("==============================================");
}
