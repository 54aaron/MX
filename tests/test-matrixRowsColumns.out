==============================================
[ 2, 3, 4 ]
[ 6, 4, 5 ]
[ 7, 8, 1 ]
[ 9, 7, 8 ]

==============================================
Number of columns for m are: 
3
==============================================
[ 1, 2, 4 ]
[ 3, 4, 5 ]

==============================================
Number of rows for n are: 
2
==============================================
